internal/idle/idle_e2e_test.go

```diff
@@ -131,12 +131,12 @@ func (s) TestChannelIdleness_Disabled_NoActivity(t *testing.T) {
 		grpc.WithIdleTimeout(0), // Disable idleness.
 		grpc.WithDefaultServiceConfig(`{"loadBalancingConfig": [{"round_robin":{}}]}`),
 	}
-	cc, err := grpc.Dial(r.Scheme()+":///test.server", dopts...)
+	cc, err := grpc.NewClient(r.Scheme()+":///test.server", dopts...)
 	if err != nil {
-		t.Fatalf("grpc.Dial() failed: %v", err)
+		t.Fatalf("Failed to create new client: %v", err)
```

Comment:
i see that in some places you replaced with grpc.NewClient() and some places you have "Failed to create new client". Please be consistent. grpc.Dial() failed can be changed to "grpc.NewClient() failed" and "Failed to dial" can be changed to "Failed to create new client"

Reply:
done please check